### PR TITLE
[MOSIP-28823] Update registration-processor-camel-routes-update-mz.xml

### PR DESCRIPTION
**Pull request by n-chandrasekhar**

### CODE REVIEW

File: sandbox/registration-processor-camel-routes-update-mz.xml [hunk 1]
```diff
@@ -67,7 +67,7 @@
             message="packet-classifier-->quality-checker update route ${bodyAs(String)}" />
         <choice>
               <when>
-                <simple>${bodyAs(String)} contains '"isValid":true'</simple>
+		<jsonpath suppressExceptions="true">$.[?(@['tags']['ID_OBJECT-presentCity'] != 'Pilor' &amp;&amp; @['isValid'] == true)]</jsonpath>
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-camel-routes-update-mz.xml]: Is this Pilar or different city ?

File: sandbox/registration-processor-camel-routes-update-mz.xml [hunk 2]
```diff
@@ -111,6 +111,7 @@
             message="osi-validator-->biometric-authentication-stage update route ${bodyAs(String)}" />
         <choice>
             <when>
+		<jsonpath suppressExceptions="true">$.[?(@['tags']['ID_OBJECT-presentCity'] != 'Tubo' &amp;&amp; @['isValid'] == true)]</jsonpath>
                 <simple>${bodyAs(String)} contains '"isValid":true'</simple>
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-camel-routes-update-mz.xml]: why simple is not removed ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: sandbox/registration-processor-camel-routes-update-mz.xml (hunk 2 above)
Comment (n-chandrasekhar) [reply sandbox/registration-processor-camel-routes-update-mz.xml]: Removed Simple and changed city Tubo to Lacub

File: sandbox/registration-processor-camel-routes-update-mz.xml (hunk 1 above)
Comment (n-chandrasekhar) [reply sandbox/registration-processor-camel-routes-update-mz.xml]: Pilar is correct. Changed to Pilar.

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**